webwhatsapi/js/wapi.js

```diff
@@ -1147,3 +1148,27 @@ window.WAPI.getBufferedNewMessages = function(done) {
     return bufferedMessages;
 };
 /** End new messages observable functions **/
+
+window.WAPI.sendImage = function (imgBase64, chatid, filename, caption, done) {
+    var chat = WAPI.getChat(chatid);
+    if (chat !== undefined) {
+        var mediaBlob = window.WAPI.base64ImageToFile(imgBase64, filename);
+        Store.MediaCollection.processFiles([mediaBlob], chat, 1).then(() => {
+            var media = Store.MediaCollection.models[Store.MediaCollection.models.length - 1];
```

Comment:
Wow! you must free memory!!!

Reply:
i don't know how clear >Store.MediaCollection.models<

Reply:
WhatsappWeb use AmpresandJS (https://ampersandjs.com/) and Reactjs (https://reactjs.org/). In front of doubt, look at documentation.